lms/static/scripts/frontend_apps/components/FileList.js

```diff
@@ -5,8 +5,27 @@ import propTypes from 'prop-types';
 import Spinner from './Spinner';
 import Table from './Table';
 
+/**
+ * @typedef File
+ * @prop {string} display_name - The filename
```

Comment:
I presume these property names are externally-imposed (i.e. why it's not `displayName`, `updatedAt`)?

Reply:
Yes, exactly. The come from the LMS frontend <-> backend API which uses Pythonic naming conventions.